### DISCUSSION TITLE
Receptionクラスの実装

### DISCUSSION THREAD

**Discussion by sankakumusic** [kkksb/python_bank#5] [Ideas]
Receptionクラスの関数はメンバ変数にアクセスしないのでスタティックにすると呼び出しが簡潔になるのではないでしょうか。
https://github.com/kkksb/python_bank/blob/main/pybank/libs/reception.py#L5

**Comment by kkksb** [comment]
Discussionってコードレビューにも使えるんですね...

そうですね、Receptionのメソッドについてはstaticにしておこうと思います。
